src/AuthModule/Components/Register/Register.tsx

```diff
@@ -68,39 +78,42 @@ export default function Register() {
     email: string;
     password: string;
     confirmPassword: string;
-    role:string;
-    profileImage:string;
+    role: string;
+    profileImage: string;
   };
 
-  const appendRegisterFormData=(data:RegisterData)=>{
-    let formData=new FormData();
-    formData.append("userName",data.userName);
-    formData.append("phoneNumber",data.phoneNumber);
-    formData.append("country",data.country);
-    formData.append("email",data.email);
-    formData.append("password",data.password);
-    formData.append("confirmPassword",data.confirmPassword);
-    formData.append("role",data.role);
-    formData.append("profileImage",data.profileImage[0]);
+  const appendRegisterFormData = (data: RegisterData) => {
+    let formData = new FormData();
+    formData.append("userName", data.userName);
+    formData.append("phoneNumber", data.phoneNumber);
+    formData.append("country", data.country);
+    formData.append("email", data.email);
+    formData.append("password", data.password);
+    formData.append("confirmPassword", data.confirmPassword);
+    formData.append("role", "user");
+    formData.append("profileImage", data.profileImage[0]);
     return formData;
-  }
+  };
   async function handleRegister(data: RegisterData) {
     console.log(data);
-    let registerDataForm= appendRegisterFormData(data);
+    let registerDataForm = appendRegisterFormData(data);
 
     try {
-      const response = await axios.post(`${baseUrl}/admin/users`, registerDataForm);
+      const response = await axios.post(
+        `${baseUrl}/admin/users`,
+        registerDataForm
+      );
       // console.log(response);
-      toast.success('Account Created Succefully');
-      navigate('/Authentication');
+      toast.success("Account Created Succefully");
+      navigate("/Authentication");
     } catch (error) {
       // console.log(error?.response?.data?.message);
-      toast.error(error?.response?.data?.message||'Somthing went wrong!');
+      toast.error(error?.response?.data?.message || "Somthing went wrong!");
     }
   }
   return (
     <>
-      {/* <ThemeProvider theme={defaultTheme}> */}
+      <ThemeProvider theme={defaultTheme}>
```

Comment:
This is not good, as we should make this theme provide in a top-level component and access it in any descendent component using the `useTheme` hook.